docs/tutorials/behavior_knowledgebase.md

```diff
@@ -0,0 +1,283 @@
+---
+icon: material/silverware-fork-knife
+---
+
+# 🍴 **BEHAVIOR Knowledgebase**
```

Comment:
In general, it would be really helpful to have a nice overview figure here. Something that visually shows the bridge between Knowledgebase and OG, e.g.: synset on one side, bridge connecting it to the OG object model(s) on the other side, with category in between, etc.

Can discuss with you tomorrow, this can be a separate PR

Reply:
True, completely agreed! This doc still has much room for improvement! I will leave this comment open.